pyrate/gdal_python.py

```diff
@@ -17,18 +17,44 @@
 This Python module contains bindings for the GDAL library
 """
 # pylint: disable=too-many-arguments,R0914
-import gc
 from osgeo import gdal, gdalnumeric, gdalconst
 from PIL import Image, ImageDraw
 import numpy as np
+import numexpr as ne
+
 from pyrate import ifgconstants as ifc
 from pyrate import shared
+from pyrate import prepifg 
 
 gdal.SetCacheMax(2**15)
 GDAL_WARP_MEMORY_LIMIT = 2**10
 LOW_FLOAT32 = np.finfo(np.float32).min*1e-10
 
-
+def coherence_masking(src_ds, coh_ds, coh_thresh):
+    """
+    Perform coherence masking on raster in-place. 
+  
+    Based on gdal_calc formula provided by Nahidul:
```

Comment:
Can you add Nahidul to docs/authors.rst as well.

Reply:
Done!